src/main/java/org/dependencytrack/resources/v1/TagResource.java

```diff
@@ -501,4 +503,77 @@ public Response untagNotificationRules(
 
         return Response.noContent().build();
     }
+
+    @GET
+    @Path("/{name}/vulnerability")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(
+            summary = "Returns a list of all vulnerabilities assigned to the given tag.",
+            description = "<p>Requires permission <strong>VULNERABILITY_MANAGEMENT</strong> or <strong>VULNERABILITY_MANAGEMENT_UPDATE</strong></p>"
```

Comment:
As per `@PermissionRequired` annotation, this endpoint requires `READ`, not `UPDATE` permission.

```suggestion
            description = "<p>Requires permission <strong>VULNERABILITY_MANAGEMENT</strong> or <strong>VULNERABILITY_MANAGEMENT_READ</strong></p>"
```